src/model_tool.py

```diff
@@ -27,6 +30,9 @@
 EMBEDDING_DIM = 100
 EMBEDDING_TRAINABLE = False
 LEARNING_RATE = 0.00005
+STOP_EARLY = True
+ES_PATIENCE = 0 # Only relevant if STOP_EARLY = True
+ES_MIN_DELTA = 0 # Only relevant if STOP_EARLY = True
```

Comment:
Are these reasonable defaults?

Reply:
Yes, I think they are until we've done further experimentation.